src/components/LibraryStageItem.tsx

```diff
@@ -15,7 +15,7 @@ export default class LibraryStageItem extends React.Component<LibraryStageItemPr
     const { label, value } = this.props;
 
     let select = (
-      <select key={label} name={label} defaultValue={value}>
+      <select key={label} name={label} defaultValue={value} aria-label={`Select ${label}`}>
```

Comment:
I think ideally we'd have an actual `label` element. I was thinking we could have used the legend for each form and use `aria-labelledby` for the select element. It's not in the same component and it complicates things. The easiest thing to make this pass is to add an `aria-label` (I also didn't know if you would've wanted to have an extra label element thrown into the UI).

Reply:
It doesn't seem worth having an extra label element for this; it would be one thing if we were trying to use the component across more interfaces, but for now...

Reply:
Yeah, it's not an extra label element, it's just the attribute. It's either the attribute or the label element and the attribute doesn't change the UI.

Reply:
No, I just meant re: what you were saying about not throwing an extra label element into the UI: good call!